lib/kitchen/driver/docker.rb

```diff
@@ -384,6 +393,17 @@ def container_ssh_port(state)
         end
       end
 
+      def container_ip(state)
+        begin
+          cmd = "inspect --format '{{ .NetworkSettings.IPAddress }}'"
```

Comment:
@amalucelli @AaronKalair FYI, there is an issue when using this setup with user defined networks.

In those cases, the IPAddress at the NetworkSettings level is empty, and instead needs to be found on the Network. That nested value is also there when using the default bridge network, so I would think that it should be safe to use in any case:
I was able to get this working by changing the template to `'{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'`